api/v1alpha1/envoyproxy_tracing_types.go

```diff
@@ -31,6 +31,8 @@ type ProxyTracing struct {
 	SamplingFraction *gwapiv1.Fraction `json:"samplingFraction,omitempty"`
 	// CustomTags defines the custom tags to add to each span.
 	// If provider is kubernetes, pod name and namespace are added by default.
+	//
```

Comment:
can we take SamplingRate, SamplingFraction and CustomTags, and put in into another struct which we can embed here and in BackendTelemetry ?

Reply:
I think we should move to use SamplingFraction and deprecate SamplingRate in the future.

RDS support FractionalPercent directly, so I don't want another convertion here.

Reply:
cool, can you create a embedded struct inside `Tracing` so `SamplingFraction` and `CustomTags` can be reused

Reply:
The reason I cannot use `Tracing` inline in `ProxyTracing` is because we need following comment:
```
	// Only one of SamplingRate or SamplingFraction may be specified.
	// If neither field is specified, all requests will be sampled.
```

Reply:
makes sense